SIL.Core/Reporting/Logger.cs

```diff
@@ -481,7 +482,19 @@ private void WriteMinorEventCore(string message, params object[] args)
 						m_minorEvents.Remove(0, cutoff);
 					}
 					m_minorEvents.Append(DateTime.Now.ToLongTimeString() + "\t");
-					m_minorEvents.AppendFormat(message, args);
+					if (args.Any())
```

Comment:
In case anybody was wondering if args could be null, Section 7.4.1 of the C# Language Specification says
> In particular, note that an empty array is created when there are zero arguments given for the parameter array.